machine-learning/01-Intro to Machine Learning.ipynb

```diff
@@ -1,402 +1,392 @@
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "dotnet_interactive": {
-          "language": "csharp"
-        },
-        "vscode": {
-          "languageId": "csharp"
-        }
-      },
-      "outputs": [],
-      "source": [
-        "var size = new HouseData() { Size = 2.5F };\n",
-        "var price = predictionEngine.Predict(size);\n",
-        "Console.WriteLine($\"Predicted price for size: {size.Size*1000} sq ft= {price.Price*100:C}k\");"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {},
-      "source": [
-        "Congrats! You have successfully trained an ML.NET regression model using your own data, then used this model to predict the house prices. Here is a diagram summarizing the end-to-end operation of creation and training ML.NET model then using it to predict the house prices.\n",
-        "\n",
-        "![](https://docs.microsoft.com/dotnet/machine-learning/media/mldotnet-annotated-workflow.png)\n"
-      ]
-    }
-  ],
-  "metadata": {
-    "kernelspec": {
-      "display_name": ".NET (C#)",
-      "language": "C#",
-      "name": ".net-csharp"
-    },
-    "language_info": {
-      "file_extension": ".cs",
-      "mimetype": "text/x-csharp",
-      "name": "C#",
-      "pygments_lexer": "csharp",
-      "version": "8.0"
-    }
-  },
-  "nbformat": 4,
-  "nbformat_minor": 4
-}
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Introduction to Machine Learning"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## What is Machine Learning?\n",
+        "\n",
+        "\n",
+        "```\n",
+        "var size = new HouseData() { Size = 2.5F };\n",
+        "var price = predictionEngine.Predict(size);\n",
+        "```\n",
+        "\n",
+        "The above code shows how to consume a model that's already been trained. The end result of training a model is a function you can pass some data `HouseData.Size`  to the model and it will give you back a prediction - `Prediction.Price`. \n",
+        "\n",
+        "The above is a simple example (probably too simple) but models can take in many more values.  For instance - [Value Prediction(Regression) with Taxi Dataset](https://raw.githubusercontent.com/dotnet/csharp-notebooks/fa302c12c7494e5f8a5fdbe5d8283d8ff1fb7009/machine-learning/E2E-Value%20Prediction(Regression)%20with%20Taxi%20Dataset.ipynb) -- \n",
+        "is a more complex example that takes in `vendor_id`,  `rate_code`, `passenger_count`, `trip_time_in_secs`, `trip_distance`, `payment_type` and then predicts `fare_amount`.\n",
+        "\n",
+        "### How do you create that function?\n",
+        "Machine Learning creates or trains that model by feeding an algorithm a bunch of data and using statistics to predict the values. More details and example below!"

```

Comment:
How nit-picky can I be? You use the phrase "a bunch" three times in the next two paragraphs.

Reply:
Jeesh I really used it a bunch.

Reply:
![](https://media.giphy.com/media/Bng9nsAhSaDVxWsSLh/giphy.gif)